map.html

```diff
@@ -1,4 +1,4 @@
----
+﻿---
```

Comment:
Something is up with this line 1 change that is causing the page to not render correctly.

Reply:
No idea.. 
replaced the top line with the --- below it.. seems to work now.
WIERD